crates/type-system/build-wasm.ts

```diff
@@ -0,0 +1,79 @@
+import * as path from "node:path";
```

Comment:
Moving this file to `crates/type-system/scripts/build-wasm.ts` should fix most of ESLint errors. You can also run `yarn fix` locally to address a few. We can look at the remaining ones afterwards.

Reply:
That was done https://github.com/blockprotocol/blockprotocol/pull/503/commits/3891af587f3efc63e1fa992637fe8056c3f0fb62

Thanks :)